crates/fluvio-cli/src/error.rs

```diff
@@ -32,10 +32,6 @@ pub enum CliError {
     #[error("Kubernetes config error: {0}")]
     K8ConfigError(#[from] k8_config::ConfigError),
```

Comment:
do we still need this one?

Reply:
I didn't update it, but it seems to compile successfully without this